docs/src/dev-contributors.md

```
@@ -0,0 +1,120 @@
+---
+title: Contributors
+description: Contributors to the KubeElasti project
+---
+
+This page recognizes all the amazing people who have contributed to the KubeElasti project. We appreciate all contributions, from code to documentation, testing, and community support.
+
+<br>
```

markdownlint-cli2 (0.17.2)

Inline HTML
Element: div

(MD033, no-inline-html)